app/src/main/res/values/strings.xml

```diff
@@ -1,81 +1,179 @@
 <resources>
+    <!-- LOCALIZATION NOTE (app_name): The string 'app_name' will be used any time the application
+         name is to be displayed to the user inside of the application as well as any system launcher
+         such as the text bellow the application launch icon. -->
```

Comment:
I don't think you need to call out "the string 'app_name', you can just say "This string will be used any time...".
Also nit: "bellow" -> "below"
(also not sure if you'll wanna use a variable like in other parts for the app name? In that case please explain what the variable stands for in the localization comment)